### PR TITLE
Pass more tests on webid-provider-tests

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
This work continues on top of #7, that's why I based this PR on that one. I already merged it into `dev` (which is the branch the test-suites look at now) but will wait for your review before merging both into `master`.

**Review by Potherca — APPROVED** [review]

**Comment by michielbdejong** [comment]
thanks! will merge this as part of #13

**Pull request closed without merging**
